src/index.ts

```diff
@@ -0,0 +1,356 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as extend from 'extend';
+import * as util from 'util';
+// TODO: Address type incorrectness in @types/is
+const is: {fn: (value: {}) => boolean; object: (value: {}) => boolean} =
+    require('is');
+const logging = require('@google-cloud/logging');
+const mapValues = require('lodash.mapvalues');
+import * as winston from 'winston';
```

Comment:
style nit: group all `import`s together at the top. Keep `requires` separate.

Reply:
Done.